components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -148,6 +153,9 @@ internal class DisplayToolbar(
             field = value
         }
 
+    // Optional callback to be call when the icon.
+    internal var onIconClicked: (() -> Unit)? = null
+
```

Comment:
This one seems unneeded?

Reply:
Sorry a leftover from some previous experiments :P

Reply:
Why are you not onboarding?? :P